### DISCUSSION TITLE
The history of sleep

### DISCUSSION THREAD

**Discussion by mimimix** [zepp-health/zeppos-forum#371] [Q&A]
There was an idea to make a small application that would show the average sleep duration for a week and a month right on the clock. But I have not found a method that would allow me to get sleep data for previous days. 
In the standard sleep app, it is also not possible to see information for previous days.
Now I only have the idea to save information about sleep duration every day and work with it later. But this is not very convenient, because it will not be possible to get information for the past days, which is available in the application on the phone.

**Comment by silver-zepp** [comment]
The feature you are asking for, to allow our devs to pull a whole week of sleep data is already being worked on and should be available with one of the next zeppos updates.
